Lun 3517

mariasumedre-3pg opened the pull request.

mariasumedre-3pg committed:
LUN-3517: fix detection of duplicate files

mariasumedre-3pg committed:
LUN-3517: do not show popup when same file uploaded

mariasumedre-3pg committed:
LUN-3517: update unit-tests for clipboardadmin after changes

alexandrupirjol commented on filer/static/filer/js/fileuploader.js:
```diff
@@ -375,10 +375,13 @@ qq.FileUploaderBasic.prototype = {
+        // the error is shown already in clipboard list
+        // and for multiple uploads, there are just as many pop-ups
+        // and that becomes annoying
+        // if (result.error) {
```
please remove commented code

mariasumedre-3pg committed:
LUN-3517: removed commented code

zoliszeredi commented on filer/tests/admin.py:
```diff
@@ -245,6 +248,8 @@ def test_file_upload_no_duplicate_files(self, extra_headers={}):
         )
         self.assertEqual(Image.objects.count(), 1)
         self.assertIn('error', response.content)
+        self.assertIn('already exists in the clipboard', response.content)
```
Nitpick. It would be really nice if the error message would be stored in a field on the admin class. 
That way this test won't break when the message changes.

zoliszeredi commented:
LGTM :+1:

mariasumedre-3pg committed:
LUN-3517: Put error messages in a dictionary

Since error messages are checked in unit-tests, to make sure the right
error is expected, put the error messages once in a dictionary in the
admin class. Then use the dictionary in both the ClipboardAdmin class
and the unit-test
